src/mjolnir/edgeinfobuilder.cc

```diff
@@ -24,10 +24,10 @@ void EdgeInfoBuilder::set_wayid(const uint64_t wayid) {
 
 // Set the mean elevation.
 void EdgeInfoBuilder::set_mean_elevation(const float mean_elev) {
-  if (mean_elev < kMinElevation) {
+  if (mean_elev <= kNoElevationData) {
```

Comment:
isnt it the case that we need to differentiate between no data and minimum value though? like when its no data then we serialize null and when its minimum value we serialize the minimum value, right?

Reply:
Minimum value (-500) will not be serialized, same as no data. Think that’s fine, there’s really no place on terrestrial earth at -500. Dead Sea depression is at -450 or so, roads near that have the lowest values in the world.

Reply:
yeah you are saying in practice -500 wont happen so no_data can be the same as min_value but like you mentioned at the beginning it does seem a bit semantically strange in the computation later where you do an offset by the no data value. i'd be happy just duplicating the constant even if they were the same value so that when you read the code you offset by the min-value rather than the no-data-value

Reply:
yeah true, wasn't very happy with that either but with the comment above the constant in graphconstants I found it ok-ish.. then let me duplicate that constant

Reply:
reverted the change to edgeinfo.h where kMinElevation was set before, which reverted all changes to edgeinfo.h and edgeinfobuilder.h. kMinElevation is now used where appropriate

is the internet broken today?! github is being ridiculously slow or doesn't respond at all after pushing, geofabrik same, curl actually times out downloading some pbf.. maybe it's telling me to go home instead..